Khan/perseus
# [Locked Figure Aria] Implement locked line aria label behavior on graph

## Summary:
- Add `ariaLabel` field to LockedLineType
- Add the aria label behavior to the LockedLine on the mafs graph

Issue: https://khanacademy.atlassian.net/browse/LEMS-2376

## Test plan:
`yarn jest packages/perseus/src/widgets/interactive-graphs/interactive-graph.test.tsx`

Storybook
- Go to http://localhost:6006/iframe.html?args=&id=perseuseditor-widgets-interactive-graph--mafs-with-locked-figure-labels-all-flags&viewMode=story
- Use a screen reader to navigate through the preview graph
- Using the screenreader controls (VO + arrow keys) should be able to navigate to
  the locked line. It should read out as "Point PQ".


https://github.com/user-attachments/assets/a6fafd5b-9d3e-4b1a-94ca-7404bb60e7ab
